data_hub_api/docmaps/v2/codecs/elife_manuscript.py

```diff
@@ -104,16 +105,20 @@ def iter_elife_manuscript_part_of_section_complement_for_one_record(
 ) -> Iterable[DocmapPartOfComplement]:
     if related_content['manuscript_id']:
         assert related_content['manuscript_type']
+        manuscript_type = related_content['manuscript_type'].lower().strip()
+        cleaned_manuscript_type = re.sub(r'[^a-z0-9 ]', ' ', manuscript_type)
+        manuscript_type_parts = re.split(r'\s+', cleaned_manuscript_type)
+        capitalized_parts = [part.capitalize() for part in manuscript_type_parts[1:]]
         yield {
-            'type': related_content['manuscript_type'],
+            'type': manuscript_type_parts[0] + ''.join(capitalized_parts),
```

Comment:
I was considering making this a util function with it's own tests but decided to just get the overall code working first. I can do that refactor if requested. Although there is no use for the code outside of this function.